src/styles/blocks/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid $color-card-border;
+  padding-inline: 16px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+  transition: transform 0.3s ease-in-out;
+
+  &:hover {
+    transform: scale(120%);
+  }
+
+  &__picture {
+    display: flex;
+    height: 134px;
+    padding-inline: 3px;
+    margin-bottom: 40px;
+  }
+
+  &__image {
+    width: 100%;
+  }
+
+  &__title {
+    @include typography(12px, 18px, 500, $color-main-accent);
+
+    margin-top: 0;
+    margin-bottom: 4px;
+    transition: color 0.3s ease-in-out;
+  }
+
+  &:hover &__title {
+    color: $color-card-title-hover;
+  }
+
+  &__subtitle {
+    @include typography(10px, 14px, 400, $color-secondary);
+
+    display: flex;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    @include typography(10px, 14px, 400, $color-main-accent);
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    & :first-child {
+      @include typography(12px, 18px, 400, $color-secondary);
+    }
+  }
+
+  &__cost {
+    @include typography(16px, 18px, 700, $color-main-accent);
+  }
+
+  &__button {
+    @include typography(14px, 16px, 700, $color-white);
+
+    text-transform: uppercase;
+    text-decoration: none;
+    text-align: center;
+
+    box-sizing: border-box;
```

Comment:
it's better to set for each element on the page (use `*` selector) instead of writing it manually everywhere
but note that it's only one property allowed to set in `*`

Reply:
Where should it be written? Is it correct to write it in main.scss?